depends/packages/qt.mk

```
@@ -131,6 +131,9 @@ $(package)_config_opts_darwin += -no-feature-corewlan
 $(package)_config_opts_darwin += -no-freetype
 $(package)_config_opts_darwin += QMAKE_MACOSX_DEPLOYMENT_TARGET=$(OSX_MIN_VERSION)
 
+$(package)_config_opts_aarch64_darwin += "QMAKE_CFLAGS_OPTIMIZE = '-O1'"
+$(package)_config_opts_aarch64_darwin += "QMAKE_CFLAGS_OPTIMIZE_FULL = '-O1'"
+
```

```suggestion
$(package)_config_opts_aarch64_darwin += "QMAKE_CFLAGS_OPTIMIZE_FULL = -O1"

```

will be enough. It changes optimization from `-O3` to `-O1` for the `corelib` and `gui` libraries only. Actually, we are interested in the latter only.

---

Also it would be nice to have a comment here.

---

Adjusted and added a comment.